tests/drum/test_custom_model.py

```diff
@@ -681,6 +692,32 @@ def test_fit(self, framework, problem, language, docker, weights, use_output, tm
             cmd, "Failed in {} command line! {}".format(ArgumentsOptions.MAIN_COMMAND, cmd)
         )
 
+    @pytest.mark.parametrize("problem", [BINARY, REGRESSION])
+    @pytest.mark.parametrize("use_output", [True, False])
+    def test_r_fit(self, problem, use_output, tmp_path):
```

Comment:
Why not just add a parameter to test_fit?

Reply:
As those params currently stand, I believe we're testing every combination of each param, which won't work if R params are introduced for language or framework.

It may be that there's an easy way I don't know about to test all the desired python combinations without testing impossible things like language=python and framework=rds?

Reply:
I think you should get rid of the language param, and just make R a framework, and then have some dumb logic to choose language within the test.

Reply:
that makes sense